mock/databaseWriterStub.go

```diff
@@ -0,0 +1,74 @@
+package mock
+
+import (
+	"bytes"
+
+	"github.com/elastic/go-elasticsearch/v7/esapi"
+)
+
+// DatabaseWriterStub --
```

Comment:
Please remove those double -- (valid for all methods)

Reply:
done.